### DISCUSSION TITLE
Issues generating certificates

### DISCUSSION THREAD

**Discussion by CodeMeAPixel** [vercel/community#5552] [Help]
### Summary

Hello there, i am experiencing some issues after adding a domain to my project where vercel is continuously attempting to create ssl certs for the domain due to this issue the domain is setup and working how ever we can not use full ssl which can be a problem. 

not sure how else to word this but essentially our domain is added and working but vercel is not generating its needed certs, it has been going on 24 hours since the initial domain setup and just wanted to see if there is possibly something i am doing wrong.

![DhFBLNSC](https://github.com/vercel/community/assets/59587139/9e0554b8-0544-436c-944c-3c4821228069)

### Example

dashboard-git-dev-bruhhwhy.vercel.app

### Steps to Reproduce

1. add a new domain in the projects "domain settings"
2. follow the docs to add a new domain via cloudflare with proxy support
3. vercel doesn't appear to generate the needed cert(s)

**Comment by kakarot-dev** [comment]
Same issue on my side!

**Reply by swarnava** [reply]
Hi, what is your domain?

**Comment by Calummullen** [comment]
Having the same issue over the weekend as well. Added the appropriate A, CNAME (for sub domain) and CAA records to my domain registrar (Godaddy) and it's been stuck on 'Generating SSL certificate' ever since. But the HTTP version works fine.

Swapped over to nameservers and just get stuck on the error to disable DNSSEC with my registrar despite them confirming it's never been enabled so just stuck either way

**Reply by swarnava** [reply]
Hi, what is your domain?

**Reply by Calummullen** [reply]
@swarnava 
`www.viewtube.uk`, `viewtube.uk` (redirects to www) and `app.viewtube.uk`.
Happy to swap back to try A + CNAME records if the nameserver approach isn't working though.

**Comment by swarnava** [comment]
Hi, I can see that beta.assistifyai.org has certificate enabled. Let us know if you are still seeing this behavior.

**Reply by CodeMeAPixel** [reply]
Strange, the dashboard itself says its still attempting to generate, and i am unable to use anything other then "flexible" ssl in cloudflare which is unusual behavior, i have other sites on vercel running full or full strict with a proxied connection but that is not the case here for some reason: 
![Zaoze7FG](https://github.com/vercel/community/assets/59587139/029b7a70-9e68-458a-bcc9-524b4c223bbc)
